test/dotnet/input/InsertBulk/insertBulk.txt

```diff
@@ -296,6 +296,110 @@ Select * from destinationTable
 drop table sourceTable
 drop table destinationTable
 
+Create table sourceTable(a int, b int identity)
```

Comment:
Add comments on top of the tests and explain their purpose

Reply:
Added comments